### DISCUSSION TITLE
MDAnalysis User Group Meeting 2024

### DISCUSSION THREAD

**Discussion by RMeli** [MDAnalysis/mdanalysis#4431] [Announcements]
**We are excited to announce that the [2024 MDAnalysis UGM](https://www.mdanalysis.org/pages/ugm2024/) will take place August 21-23, 2024 in London, UK!**

Additional details about registration and a call for proposals will be communicated soon. In the meantime, save the date and keep an eye out for additional announcements on our [blog](https://www.mdanalysis.org/blog/) and [UGM website page](https://www.mdanalysis.org/pages/ugm2024/), as well as on our [X](https://twitter.com/mdanalysis) and [LinkedIn](https://www.linkedin.com/company/mdanalysis/) accounts.

---

* [UGM 2024](https://www.mdanalysis.org/pages/ugm2024/)
* [Save the Date for the 2024 MDAnalysis UGM](https://www.mdanalysis.org/2024/01/15/mda_events/#save-the-date-for-the-2024-mdanalysis-ugm)
* [MDAnalysis Events - Recap of 2023 and Looking Ahead to 2024](https://www.mdanalysis.org/2024/01/15/mda_events/#mdanalysis-events-recap-of-2023-and-looking-ahead-to-2024)

**Comment by orbeckst** [comment]
Abstracts and Bursary deadline is **March 15**!

See https://www.mdanalysis.org/pages/ugm2024/#abstracts-and-registration

**Comment by jennaswa** [comment]
The abstract and bursary application deadline as now been extended to **April 15, 2024**! We are also excited to announce 2 fantastic keynote speakers, Drs. Antonia Mey and Francesca Stanzione! Read more on the MDAnalysis blog: https://www.mdanalysis.org/2024/03/14/ugm2024-updates/.
